app/src/common/shared/com/igalia/wolvic/ui/widgets/KeyboardWidget.java

```diff
@@ -1182,7 +1182,7 @@ private void postDisplayCommand(Runnable aRunnable) {
         }
     }
 
-    private void updateCandidates() {
+    private void updateCandidates(boolean resetComposingText) {
```

Comment:
Please do replace the boolean by an enumerated type which would not require developers to check the signature of the method to understand what it does. You can create something like
```
enum ResetComposingText {
YES,
NO
};
```
and then call sites would have a much more descriptive look